### PR TITLE
openapiv3: update 3.1.0.md to latest release

### PR DESCRIPTION
**Pull request by zchee**
Update `openapiv3/schema-generator/3.1.0.md` to the latest release.
copied from https://github.com/OAI/OpenAPI-Specification/blob/3.1.0/versions/3.1.0.md.
And, fix `openapiv3/schema-generator/main.go` which openapiv3 markdown syntax change.

Also, regen all `.pb.go`

### CODE REVIEW

**Comment by zchee** [comment]
@timburks Sorry just ping 🙏

File: openapiv3/OpenAPIv3.pb.go [hunk 1]
```diff
@@ -1623,8 +1623,8 @@ type Info struct {
 	Contact                *Contact    `protobuf:"bytes,4,opt,name=contact,proto3" json:"contact,omitempty"`
 	License                *License    `protobuf:"bytes,5,opt,name=license,proto3" json:"license,omitempty"`
 	Version                string      `protobuf:"bytes,6,opt,name=version,proto3" json:"version,omitempty"`
-	SpecificationExtension []*NamedAny `protobuf:"bytes,7,rep,name=specification_extension,json=specificationExtension,proto3" json:"specification_extension,omitempty"`
-	Summary                string      `protobuf:"bytes,8,opt,name=summary,proto3" json:"summary,omitempty"`
+	Summary                string      `protobuf:"bytes,7,opt,name=summary,proto3" json:"summary,omitempty"`
+	SpecificationExtension []*NamedAny `protobuf:"bytes,8,rep,name=specification_extension,json=specificationExtension,proto3" json:"specification_extension,omitempty"`
```
Comment (fenollp) [inline openapiv3/OpenAPIv3.pb.go:1627]: Shouldn't field ids changes be forbidden? This is a breaking api change.

Comment (zchee) [reply openapiv3/OpenAPIv3.pb.go:1627]: @fenollp

Are you means above?

```starlark
	SpecificationExtension []*NamedAny `protobuf:"bytes,7,rep,name=specification_extension,json=specificationExtension,proto3" json:"specification_extension,omitempty"`
	Summary                string      `protobuf:"bytes,8,opt,name=summary,proto3" json:"summary,omitempty"`
```

to

```starlark
	Summary                string      `protobuf:"bytes,7,opt,name=summary,proto3" json:"summary,omitempty"`
	SpecificationExtension []*NamedAny `protobuf:"bytes,8,rep,name=specification_extension,json=specificationExtension,proto3" json:"specification_extension,omitempty"`
```

?

**Comment by zchee** [comment]
@timburks @fenollp 
Hi! any opinions for this change?

**Comment by timburks** [comment]
@zchee Thanks for this - as @fenollp pointed out, reordering the proto fields is a breaking change, so I've been manually reordering them after rerunning the generator. I can make that change and merge this (in the next day or two).

**Comment by zchee** [comment]
@timburks will do

**Comment by zchee** [comment]
@timburks Done. PTAL.

**Review by timburks — APPROVED** [review]
@zchee perfect! thank you very much!

**Pull request merged**